cfme/tests/containers/test_breadcrumbs.py

```diff
@@ -48,18 +48,25 @@ def test_breadcrumbs(provider, appliance, soft_assert):
 
     for data_set in TESTED_OBJECTS:
 
-        inst = (provider if data_set.obj is ContainersProvider
-                else data_set.collection_obj(appliance).get_random_instances().pop())
-        view = navigate_to(inst, 'Details')
+        instances = ([provider] if data_set.obj is ContainersProvider
```

Comment:
**REQUIRED:** I wasn't going to say anything on this, but actually, now that there are changes, I think we could make this a little more efficient and take advantage of stuff that's already in place.

```python
TESTED_OBJECTS = ['container_services', 'container_routes',...'containers_providers']

for data_set in TESTED_OBJECTS:
    instances = ([provider] if data_set == 'containers_providers' else getattr(appliance.collections, data_set).all())
```

Now we get to get rid of ```DataSet``` object entirely.